client/my-sites/plugins/plugins-browser/index.jsx

```diff
@@ -35,7 +35,7 @@ import {
 	useWPORGInfinitePlugins,
 } from 'calypso/data/marketplace/use-wporg-plugin-query';
 import PageViewTracker from 'calypso/lib/analytics/page-view-tracker';
-import UrlSearch from 'calypso/lib/url-search';
+import useUrlSearch from 'calypso/lib/url-search/use-url-search';
```

Comment:
Are you planning to migrate all usages to use the hook (and a related HoC if used in a class component) eventually?

Reply:
Yes, after this PR, I'll make a P2 about this change and schedule migrate all usages to the new hook

Reply:
+1 marketplace first then follow up for the rest (with wider review)

Reply:
Incremental migration always sounds good 👍  Thanks